sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtQueryBuilder.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+using Azure.DigitalTwins.Core.QueryBuilder;
+
+namespace Azure.DigitalTwins.Core.Queries.QueryBuilders
+{
+    /// <summary>
+    /// Used to instantiate an ADT query.
```

Comment:
not really, maybe:
`Azure DigitalTwins Query builder that facilitates writing queries against ADT instances.`